project1-sps/script.js

```diff
@@ -67,21 +57,17 @@ var main = function (input) {
   console.log(`The number of wins is ${win}`);
   console.log(`The number of draws is ${draw}`);
   console.log(`The number of losses is ${lose}`);
```

Comment:
Console logs should be removed from code submission for cleanliness in the future